.rubocop.yml

```diff
@@ -11,7 +11,7 @@ LineLength:
 # Increase allowed lines in a method.  Short methods are good, but 10 lines
 #   is a bit too low.
 MethodLength:
-  Max: 40
+  Max: 60
```

Comment:
this was to get around this method being to bing
https://github.com/rackspace-cookbooks/nodestack/blob/add-nginx/test/unit/spec/spec_helper.rb#L29-L138

I thought it was more intuitive to keep the attributes all together in there, rather than pul them out into their own method to stasify the rule.  I'm happy to do that if you think it's cleaner.